bigtable/client/table_admin.h

```diff
@@ -164,6 +164,18 @@ class TableAdmin {
    */
   void DropAllRows(std::string table_id);
 
+  /**
+   * Get information about a single snapshot.
+   *
+   * @param snapshot_name the name of the snapshot.
+   * @param cluster the cluster name to which snapshot is associated.
+   * @return the information about the snapshot.
+   * @throws std::exception if the information could not be obtained before the
+   *     RPC policies in effect gave up.
+   */
```

Comment:
As we move towards beta, we need to create an sample snippet for each API, and link it from here, please see #393.  No need to do it in this PR, but please create a bug for it.